newIDE/app/src/UI/DrawerTopBar.js

```diff
@@ -43,9 +50,11 @@ export default (props: Props) => {
             <Close />
           </IconButton>
         )}
-        <Typography variant="h6" style={styles.title}>
-          {props.title}
-        </Typography>
+        <Tooltip title={props.title} placement="bottom" enterDelay={500}>
```

Comment:
Let's use 300ms as the delay, as done for the optional tooltip on icon buttons: 
 https://github.com/4ian/GDevelop/blob/2d899d7c52590da0f4ffab392d2f4b9a82e957d7/newIDE/app/src/UI/IconButton.js#L62-L71
 
 If you want to go the extra mile, this constant `tooltipEnterDelay` can be exported in a `UI/Tooltip.js` file to ensure consistency should we tweak this in the future. 

(I'm hesitant to say to abstract the entire material-ui `Tooltip` component in a separate file, because its usage is very much dependent on the context, dependent on material-ui, and because `newIDE/app/src/UI/DrawerTopBar.js` is already in the UI folder, so it has the "right" to use material-ui's Tooltip directly).

Reply:
Ok, thanks for this criterium for being in the UI folder or not. For now, I will just define a constant for the delay.